scib/integration.py

```diff
@@ -1,24 +1,24 @@
-#!/bin/env python
-
 ### D. C. Strobl, M. Müller; 2019-07-23
```

Comment:
might be worth adding a few more names to this list?

Reply:
This is just the integration module. We have the package author list in the `setup.cfg`